osquery/tables/system/windows/registry.cpp

```diff
@@ -72,6 +72,21 @@ void explodeRegistryPath(const std::string& path,
   rKey = osquery::join(toks, kRegSep);
 }
 
+Status sanitizeRegistryStrings(const DWORD dataType,
+                               BYTE* dataBuff,
+                               DWORD dataSize) {
+  auto status = Status(0, "OK");
+  if (dataBuff != NULL && dataSize != NULL) {
```

Comment:
What if here instead of `dataBuff != nullptr && dataSize != nullptr` we invert the logic and exit early? Something like `if (dataBuff == nullptr || dataSize == nullptr) { return Status(1, "Buffer or Size was null"); }`?